### PR TITLE
e2e_tests: add tests for ExecResource output

### PR DESCRIPTION
**Pull request by adjackura**
Use public generated proto libraries

### CODE REVIEW

File: e2e_tests/test_suites/ospolicies/ospolicies.go [hunk 1]
```diff
@@ -244,7 +244,7 @@ func testCase(ctx context.Context, testSetup *osPolicyTestSetup, tests chan *jun
 			go func() {
 				defer wg.Done()
 				logger.Printf("Rerunning TestCase %q", rerunTC.Name)
-				runTest(ctx, rerunTC, testSetup, logger)
+				//runTest(ctx, rerunTC, testSetup, logger)
```
Comment (ryanwe) [inline e2e_tests/test_suites/ospolicies/ospolicies.go:247]: Did you mean to remove this?

File: e2e_tests/test_suites/ospolicies/ospolicies_test_data.go [hunk 2]
```diff
@@ -1399,6 +1399,50 @@ func buildLinuxExecResourceTests(name, image, pkgManager, key string) *osPolicyT
 									},
 								},
 							},
+							{
+								Id: "exec-output",
+								ResourceType: &osconfigpb.OSPolicy_Resource_Exec{
+									Exec: &osconfigpb.OSPolicy_Resource_ExecResource{
+										Validate: &osconfigpb.OSPolicy_Resource_ExecResource_Exec{
+											Source: &osconfigpb.OSPolicy_Resource_ExecResource_Exec_Script{
+												Script: "if ls $1 >/dev/null; then\nexit 100\nfi\nexit 101",
```
Comment (ryanwe) [inline e2e_tests/test_suites/ospolicies/ospolicies_test_data.go:1408]: Do you expect 101 will ever be returned? If not, would `exit 100` be clearer?

File: e2e_tests/test_suites/ospolicies/ospolicies_test_data.go [hunk 3]
```diff
@@ -1399,6 +1399,50 @@ func buildLinuxExecResourceTests(name, image, pkgManager, key string) *osPolicyT
 									},
 								},
 							},
+							{
+								Id: "exec-output",
+								ResourceType: &osconfigpb.OSPolicy_Resource_Exec{
+									Exec: &osconfigpb.OSPolicy_Resource_ExecResource{
+										Validate: &osconfigpb.OSPolicy_Resource_ExecResource_Exec{
```
Comment (ryanwe) [inline e2e_tests/test_suites/ospolicies/ospolicies_test_data.go:1406]: Is `validate` required? WDYT about a test case with only `enforce`?

File: e2e_tests/test_suites/ospolicies/ospolicies_test_data.go (hunk 2 above)
Comment (adjackura) [reply e2e_tests/test_suites/ospolicies/ospolicies_test_data.go:1408]: 101 is returned during check state, then enforce runs, then 100 is returned on post check state.

**Review by ryanwe — APPROVED** [review]

**Pull request merged**